external-crates/move/crates/move-compiler/src/typing/translate.rs

```diff
@@ -3320,6 +3351,23 @@ fn resolve_exp_dotted(
             edotted.loc = eloc;
             borrow_exp_dotted(context, mut_, edotted)
         }
+    };
+
+    // Even if for_autocomplete is set, we process the inner dot path to report any additional
+    // errors that may detect or report.
+    if for_autocomplete {
+        let Some(tn) = type_to_type_name(context, &edotted_ty, eloc, "autocompletion".to_string())
+        else {
+            return make_error_exp(context, eloc);
+        };
+        let mut possible_methods = context.find_all_methods(&tn);
+        let mut possible_fields = context.find_all_fields(&tn);
+        possible_methods.append(&mut possible_fields);
```

Comment:
Should we mark which is a method or field? I think it would make rendering a signature a bit easier?